lib/rules/require-spaces-in-anonymous-function-expression.js

```diff
@@ -3,8 +3,9 @@
  *
  * Type: `Object`
  *
- * Values: `"beforeOpeningRoundBrace"` and `"beforeOpeningCurlyBrace"` as child properties.
- * Child properties must be set to `true`.
+ * Values: `"beforeOpeningRoundBrace"` and `"beforeOpeningCurlyBrace"` as child
```

Comment:
We started to clean-up docs a little, could you follow the same style as in - https://github.com/jscs-dev/node-jscs/pull/1667/files

?